### ISSUE TITLE
[Bug]: Accessibility: Missing accessibility name in password share button

### ISSUE THREAD

**Issue by mpivchev** [nextcloud/server#59331]
### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [x] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

<img width="214" height="107" alt="Image" src="https://github.com/user-attachments/assets/13b11955-3303-42c6-9822-44ef2c3d3513" />

Log in button should have an accessibility name.

### Steps to reproduce

1. Open a password share

### Expected behavior

Log in button should have an accessibility name.

### Nextcloud Server version

32

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```json

```

### List of activated Apps

```shell

```

### Nextcloud Signing status

```shell

```

### Nextcloud Logs

```json

```

**Comment by susnux** [comment]
Where is this button located? Its not from the login page?

**Comment by mpivchev** [comment]
Seems to be related to password share

**Comment by susnux** [comment]
FYI this only applies to Nextcloud 33 and before thus only needs to be fixed in 33 and 32.

Already fixed in Nextcloud 34+.

**Comment by susnux** [comment]
https://github.com/nextcloud/server/pull/64196
